scripts/globals/job_utils/warrior.lua

```diff
@@ -0,0 +1,126 @@
+-----------------------------------
+-- Warrior Job Utilities
+-----------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+-----------------------------------
+xi = xi or {}
+xi.job_utils = xi.job_utils or {}
+xi.job_utils.warrior = xi.job_utils.warrior or {}
+
+-----------------------------------
+-- Ability Check Functions
+-----------------------------------
+xi.job_utils.warrior.checkAggressor = function(player, target, ability)
+    return 0, 0
```

Comment:
### That looks like returning zero with extra steps

![image](https://user-images.githubusercontent.com/6871475/115105444-f011ae00-9f2c-11eb-8bf1-de33990a15d6.png)

Reply:
It doesn't make sense to me to roll these into helper functions if all we're going to do is the exact same return of two zeroes.

Reply:
If we're going to continue to follow this pattern, I would suggest creating a single function for a "no check" scenario.

Reply:
I dunnow. Creating a new function for "always allow" would still require to edit the individual scripts and create a new check function in case someone wants to edit/add/whatever the check.
Having said that, if you guys think I should just leave the 0,0 checks in the skill script, I'll push the change, no further questions asked. Just say tge word

Reply:
I think the rule should be if we do any sort of common logic, ship it over to the global. But if all we're doing is a simple return without even a condition to it, there's no need for the extra callback to a ghlobal before doing so.